hack/choco/update-choco-package.sh

```diff
@@ -54,11 +59,14 @@ rm -fv hack/choco/tools/chocolateyinstall.ps1-e
 version="${version:1}"
 sed -i -e "s/\(<version>\).*\(<\/version>\)/<version>""${version}""\<\/version>/g" hack/choco/tanzu-community-edition.nuspec
 rm -fv hack/choco/tanzu-community-edition.nuspec-e
- 
+
+sed -i -e "s/\(\$checksum64 =\).*/\$checksum64 = ""'${windows_amd64_shasum}'""/g" hack/choco/tools/chocolateyinstall.ps1 
+rm -fv hack/choco/tools/chocolateyinstall.ps1-e
+
 git add hack/choco/tools/chocolateyinstall.ps1
 git add hack/choco/tanzu-community-edition.nuspec
  
-git commit -m "auto-generated - update tce choco install scripts for version ${version}"
+git commit -s -m "auto-generated - update tce choco install scripts for version ${version}"
```

Comment:
We dont allow merging directly into main. You must create a PR and then use the `--admin` override for the merge that PR. Please see:
https://github.com/vmware-tanzu/community-edition/blob/main/hack/release/cut-release.sh#L150-L158

Also, you shouldn't assume that you are working off of main. You need to detect which branch this commit is coming from. You can find an example here:
https://github.com/vmware-tanzu/community-edition/blob/main/hack/release/cut-release.sh#L139-L146

Reply:
Sure @dvonthenen I will add this thing.

Reply:
Thanks. If we need rollback the change for some reason, the PR method gives us the ability to just revert the change in the PR through a click of the button.

Reply:
Still waiting for this to generate a PR instead of directly committing to the branch.

Reply:
We are too close to the release. Disabling this workflow until this issue is addressed and we can merge the PR.